boards/arm/nucleo_f411re/doc/nucleof411re.rst

```diff
@@ -116,6 +117,10 @@ Default Zephyr Peripheral Mapping:
 - PWM_2_CH1 : PA0
 - USER_PB : PC13
 - LD2 : PA5
+- I2C1_SDA : PB9
+- I2C1_SCL : PB8
+- I2C2_SDA : PB2
```

Comment:
PB3?

Reply:
Done. Thought I might as well add I2C3 while I'm at it too